### ISSUE TITLE
doit:(a=->, b=->)-> 'multiple functions as default parameters'

### ISSUE THREAD

**Issue by chadams** [jashkenas/coffeescript#1043]
## multiple default values work

```
obj = ->
  doit:(a=1, b=2)->
    say()
```
## functions as default values work

```
obj = ->
  doit:(a=->)->
    say()
```
## multiple functions as default parameters throw (PARSE ERROR)

```
obj = ->
  doit:(a=->, b=2)->
    say()
```

**Comment by GeoffreyBooth** [comment]
Closing this issue as it doesn’t appear to be affecting enough people to warrant attention. If someone wants to do further investigation into this and can devise a fix, a pull request is welcome.

**Comment by GeoffreyBooth** [comment]
Fixed by #4588.
